### ISSUE TITLE
how to attach nodes to bones?

### ISSUE THREAD

**Issue by supagu** [sanja-sa/gddragonbones#23]
I have animated a character and I want to put a weapon in his hand. How can I get the hand as a node to attach my weapon node to?

**Comment by Ranoller** [comment]
I don't think you can get a bone like a node. you can use an anination player in godot and follow the animation to do that.
